windows/src/unit-tests/package-info/PackageInfoTest.pas

```diff
@@ -0,0 +1,299 @@
+unit PackageInfoTest;
+
+interface
+
+uses
+  DUnitX.TestFramework,
+
+  PackageInfo,
+  kpsfile;
+
+type
+  [TestFixture]
+  TPackageInfoTest = class(TObject)
+  private
+    DataPath: string;
+    procedure DoCompare(f1, f2: string);
+    procedure DoCompareKpsFiles(f1, f2: string; k1, k2: TKpsFile);
+
+  public
+    [Setup]
+    procedure Setup;
+
+    [TearDown]
+    procedure TearDown;
+
+    [Test]
+    procedure TestXMLInOut;
+
+    [Test]
+    procedure TestJSONInOut;
+
+    [Test]
+    procedure TestRoundTrip;
+
+    [Test]
+    procedure TestRoundTripInf;
+
+    [Test]
+    procedure TestInfInOut;
+
+    [Test]
+    procedure TestAssign;
+  end;
```

Comment:
What are your thoughts on making another test that verifies `LoadIni`, `LoadXml`, and `LoadJSON` give you identical structures? That way, when we update the packageInfo structure, we'll have a test that Developer is keeping all 3 formats in sync.

Reply:
+1

Reply:
That's pretty well tested with the round trip tests, no?